static/app/components/jsonBlock.tsx

```diff
@@ -0,0 +1,31 @@
+import {useMemo} from 'react';
+import {chromeDark, chromeLight, ObjectInspector} from 'react-inspector';
+
+import ConfigStore from 'sentry/stores/configStore';
+
+type Props = {
+  data: any;
+};
+
+function JSONBlock({data}: Props) {
+  const isDark = ConfigStore.get('theme') === 'dark';
+
+  const INSPECTOR_THEME = useMemo(
+    () => ({
+      ...(isDark ? chromeDark : chromeLight),
+      BASE_BACKGROUND_COLOR: 'none',
+      OBJECT_PREVIEW_OBJECT_MAX_PROPERTIES: 1,
+    }),
+    [isDark]
+  );
+
+  return (
+    <ObjectInspector
+      data={data}
+      // @ts-expect-error
+      theme={INSPECTOR_THEME}
+    />
+  );
+}
+
+export default JSONBlock;
```

Comment:
nit: shall we make this a named export ? :)

Reply:
I'd defer to a linter for stuff like this.

A linter could prevent default exports in all places
Or, it could require default exports in all places, and at the same time ensure that import statements use the same var name as the file being imported.